db/seeds.rb

```diff
@@ -122,7 +122,7 @@ def create_users
     100.times do
       user = User.create!(
                             username:  Faker::Internet.user_name,
-                            password:  Faker::Internet.password,
+                            password_digest:  Faker::Team.state,
                             cell:      1112223333,
                             email:     Faker::Internet.email,
```

Comment:
Why was this changed to Team? I think for clarity we should keep this as internet password